packages/admin/src/table/filterbar/filterBarPopoverFilter/FilterBarPopoverFilter.tsx

```diff
@@ -49,14 +48,16 @@ function PopoverFilter({
                 {({ form, values, handleSubmit, dirtyFields }) => {
                     const countValue = calcNumberDirtyFields(values, form.getRegisteredFields());
                     return (
-                        <div className={clsx(classes.fieldBarWrapper, countValue > 0 && classes.fieldBarWrapperWithValues)}>
-                            <div className={classes.fieldBarInnerWrapper} onClick={handleClick}>
-                                <div className={clsx(classes.labelWrapper, countValue > 0 && classes.labelWrapperWithValues)}>
-                                    <Typography variant="body1">{label}</Typography>
-                                </div>
-                                <FilterBarActiveFilterBadgeComponent countValue={countValue} />
-                                <ChevronDown />
-                            </div>
+                        <div className={classes.fieldBarWrapper}>
+                            <FilterBarButton
+                                openPopover={open}
+                                countValue={countValue}
+                                onClick={handleClick}
+                                dirtyFieldsBadge={dirtyFieldsBadge}
+                                endIcon={<ChevronDown />}
```

Comment:
I think icons should always be overridable. 

Maybe add `filterBarButtonIcon: React.ReactNode` or `filterBarButtonProps: FilterBarButtonProps` to `FilterBarPopoverFilterProps`?

Or maybe it makes sense, to set `<ChevronDown />` as the default value for `endIcon` in `FilterBarButton`?

Reply:
> Maybe add filterBarButtonIcon: React.ReactNode or filterBarButtonProps: FilterBarButtonProps to FilterBarPopoverFilterProps?

I added a `openPopoverButtonProps` prop

> Or maybe it makes sense, to set <ChevronDown /> as the default value for endIcon in FilterBarButton?

Yes that makes sense. I added that.